xml/depl_nodes.xml

```diff
@@ -3067,22 +3067,22 @@ host1:/srv/nfs/share1 /mnt/nfs/share1 -o rsize=8192,wsize=8192,timeo=14,intr
   </bridgehead>
 
   <para>
-   This plug-in requires to configure access to the &vmware; NSX service.
+   This plug-in is required for configuring access to the &vmware; service.
```

Comment:
This whole section is only for NSX so it should be taken out.  I noticed that this stuff crept into the Cloud 7 docs as well not sure how that happened.